core/src/main/java/com/oxeanbits/forecastchart/core/ui/component/CombinedChartComponent.kt

```diff
@@ -53,6 +84,36 @@ class CombinedChartComponent(context: Context) : LinearLayout(context), Anvil.Re
         }
     }
 
+    private fun loadChartDetails(data: Line){
+        linearLayout{
+            size(WRAP, WRAP)
+            orientation(HORIZONTAL)
+            textView {
+                size(WRAP, WRAP)
+                val lastData = data.values[data.values.lastIndex].y
```

Comment:
Is not possible to have some NPE here or indexOufBounds?

Reply:
You got a point! The value array can't be null but it can actually be empty. I will fix this.